src/test/java/com/leverx/learningmanagementsystem/controller/CourseControllerTest.java

```diff
@@ -0,0 +1,86 @@
+package com.leverx.learningmanagementsystem.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.leverx.learningmanagementsystem.dto.course.CreateCourseDto;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+public class CourseControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    private static CreateCourseDto createCourseDto;
+
+    @BeforeAll
```

Comment:
it's not recommended to share some data across different tests, so it's better either use `@BeforeEach` annotation or create test data inside of testing method